source/includes/_resource.collections.md

```diff
@@ -151,39 +151,113 @@ $ curl -X POST "https://api.vhx.tv/collections" \
       </td>
       <td>A publicly accessible image URL. If you prefer, you can upload images directly to a collection in the Vimeo OTT Publisher Admin.</td>
     </tr>
-    <tr class="is-internal text-2 border-bottom border--light-gray">
+    <tr class="text-2 border-bottom border--light-gray">
       <td>
         <strong class="is-block text--navy">metadata</strong>
```

Comment:
General comment that metadata should be moved down in the argument lists as standard metadata is the preferred option at this point. In fact, should we even be including this param?